subscriptions/subscriptions-impl/src/test/java/com/duckduckgo/subscriptions/impl/RealSubscriptionsTest.kt

```diff
@@ -197,23 +200,29 @@ class RealSubscriptionsTest {
 
     @Test
     fun whenLaunchPrivacyProWithOriginThenPassTheOriginToActivity() = runTest {
-        whenever(globalActivityStarter.startIntent(any(), any<SettingsScreenNoParams>())).thenReturn(mock())
+        whenever(globalActivityStarter.startIntent(any(), any<SettingsScreenNoParams>())).thenReturn(fakeIntent())
+        whenever(globalActivityStarter.startIntent(any(), any<SubscriptionsWebViewActivityWithParams>())).thenReturn(fakeIntent())
 
-        val captor = argumentCaptor<SubscriptionsWebViewActivityWithParams>()
+        val captor = argumentCaptor<ActivityParams>()
         subscriptions.launchPrivacyPro(context, "https://duckduckgo.com/pro?origin=test".toUri())
 
         verify(globalActivityStarter, times(2)).startIntent(eq(context), captor.capture())
-        assertEquals("test", captor.lastValue.origin)
+        assertEquals("test", (captor.lastValue as SubscriptionsWebViewActivityWithParams).origin)
     }
 
     @Test
     fun whenLaunchPrivacyProWithNoOriginThenDoNotPassTheOriginToActivity() = runTest {
-        whenever(globalActivityStarter.startIntent(any(), any<SettingsScreenNoParams>())).thenReturn(mock())
+        whenever(globalActivityStarter.startIntent(any(), any<SettingsScreenNoParams>())).thenReturn(fakeIntent())
+        whenever(globalActivityStarter.startIntent(any(), any<SubscriptionsWebViewActivityWithParams>())).thenReturn(fakeIntent())
 
-        val captor = argumentCaptor<SubscriptionsWebViewActivityWithParams>()
+        val captor = argumentCaptor<ActivityParams>()
         subscriptions.launchPrivacyPro(context, "https://duckduckgo.com/pro".toUri())
 
         verify(globalActivityStarter, times(2)).startIntent(eq(context), captor.capture())
-        assertNull(captor.lastValue.origin)
+        assertNull((captor.lastValue as SubscriptionsWebViewActivityWithParams).origin)
+    }
+
+    private fun fakeIntent(): Intent {
+        return Intent().also { it.addFlags(FLAG_ACTIVITY_NEW_TASK) }
```

Comment:
needed otherwise `ContextCompat.startActivities` complains about the activity being started outside of an activity context. (in production, it's already using an activity context so no changes needed to prod code)